app/views/course/assessment/submission/submissions/_submission.json.jbuilder

```diff
@@ -30,6 +30,8 @@ json.submission do
     json.grade submission.grade.to_f
   end
   json.maximumGrade assessment.maximum_grade.to_f
+  
+  json.showPublicTestCasesOutput current_course.enable_public_test_cases_output || current_course_user.staff?
```

Comment:
again, is the `staff?` check necessary here?

Reply:
Will remove this extra check as well.